docs/anomaly-detection/presets/README.md

```diff
@@ -0,0 +1,42 @@
+---
+sort: 1
+weight: 1
+title: Presets
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Presets
+---
+# Anomaly Detection Presets
+> Please check the [Quick Start Guide](/anomaly-detection/quickstart/) to install and run `vmanomaly`
+
+> Presets are available from v1.13.0
+
+Presets enable anomaly detection in indicators that are hard to monitor using alerts based on static thresholds.
+So, the anomaly detection alerting rules based on the [`anomaly_scores`](https://docs.victoriametrics.com/anomaly-detection/faq/#what-is-anomaly-score) stay the same over time, and we generate the anomaly scores using predefined machine learning models.
+Models are constantly retraining on different time frames which helps to keep alerts up to date and to consider underlying data patterns.
+
+You can set up the simplified configuration file for `vmanomaly` just specifying the type of preset and data sources in [`reader`](https://docs.victoriametrics.com/anomaly-detection/components/reader/) and [`writer`](https://docs.victoriametrics.com/anomaly-detection/components/writer/) sections of the config.
+The rest of the parameters are already set up for you.
```

Comment:
the second usecase is missing 

that people can use this as sandbox/playground or as a starting point when they need to write own metrics (i guess this should be added to some high level documentation

Reply:
Added to Overview page